src/components/Button/Button.md

```diff
@@ -3,6 +3,9 @@ Buttons are used to signal actions, submit forms, or trigger new content within
 View the [Rivet documentation for Buttons](https://rivet.uits.iu.edu/components/forms/buttons/).
 
 ### Button Examples
+
+<!-- prettier-ignore-start -->
```

Comment:
There should also be a corresponding `<!-- prettier-ignore-end -->` for each start comment.

Reply:
Woops! Thanks, revised. I hope it worked better this time...